.github/workflows/no-incorrect-image.yml

```diff
@@ -0,0 +1,17 @@
+name: I2::CI::check_for_incorrect_images
+
+on: workflow_dispatch
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Python 3.11
+        uses: actions/setup-python@v1
+        with:
+          python-version: "3.11"
+      - uses: actions/checkout@v3
+      - name: Install depenencies
+        run: pip install -r .github/scripts/ci_test/requirements.txt --no-input
+      - name: Check containers
+        run: python .github/scripts/ci_test/ci_image_scan.py
```

Comment:
I believe that in a flexible design, the script itself should not be aware of the file patterns or the exact image it should ban.

IMO, in a flexible design the script invocation should look like:

```shell
python check-compose-image.py -- docker-compose.yml --allow iroha2:stable
```

- The script doesn't care about which `.yaml` it should check
- The script doesn't know about image names, but only compares the content with the passed argument
- Multi-branch support: in `iroha2-dev` branch we could ensure that our compose files contain `iroha2:dev` image, while in `iroha2-stable` branch they contain `iroha2:stable` image. It could be achieved by using e.g. [`${{ github.ref_name }}` syntax](https://docs.github.com/en/actions/learn-github-actions/expressions#example).